pkgs/by-name/im/immich/package.nix

```diff
@@ -166,13 +166,6 @@ buildNpmPackage' {
     # see https://github.com/immich-app/immich/issues/13971
     substituteInPlace src/services/backup.service.ts \
       --replace-fail '`/usr/lib/postgresql/''${databaseMajorVersion}/bin/pg_dumpall`' '`pg_dump`'
-
-    # some part of the build wants to use un-prefixed binaries. let them.
-    mkdir -p $TMP/bin
-    ln -s "$(type -p ${stdenv.cc.targetPrefix}pkg-config)" $TMP/bin/pkg-config
```

Comment:
Why was this line removed? The build succeeds with it

Reply:
You should only have removed the two lines which caused the build to fail

Reply:
According to @rhelmot's review above, the code I removed was fixing a build failure when cross compiling. But as cross builds still work, I don't see a reason to keep the code around. I could split it up into two commits, in case we decide to revert it again in the future

Reply:
I see, thanks for the clarification.